### PR TITLE
✨quiz: Enhance accessibility and focus styles for interactive elements

### PR DESCRIPTION
**Pull request by b-l-i-n-d**

### CODE REVIEW

File: assets/react/v3/entries/course-builder/components/curriculum/question-types/ImageAnswering.tsx [hunk 1]
```diff
@@ -161,37 +187,12 @@ const ImageAnswering = () => {
...
-          if (validationError?.type === 'add_option') {
-            setValidationError(null);
-          }
-        }}
-        css={styles.addOptionButton}
-      >
-        <SVGIcon name="plus" height={24} width={24} />
-        {__('Add Option', 'tutor')}
-      </button>
+      <div>
```
Comment (sazedul-haque) [inline assets/react/v3/entries/course-builder/components/curriculum/question-types/ImageAnswering.tsx:190]: Why not passing the icon as props. In that case no need to write any extra css

File: assets/react/v3/entries/course-builder/components/curriculum/question-types/Matching.tsx [hunk 2]
```diff
@@ -165,37 +191,12 @@ const Matching = () => {
...
-            setValidationError(null);
-          }
-        }}
-        css={styles.addOptionButton}
-      >
-        <SVGIcon name="plus" height={24} width={24} />
-        {__('Add Option', 'tutor')}
-      </button>
+      <div>
+        <Button variant="text" onClick={handleAddOption} buttonContentCss={styles.addOptionButton}>
```
Comment (sazedul-haque) [inline assets/react/v3/entries/course-builder/components/curriculum/question-types/Matching.tsx:195]: Same here

File: assets/react/v3/entries/course-builder/components/curriculum/question-types/MultipleChoiceAndOrdering.tsx [hunk 3]
```diff
@@ -234,39 +260,12 @@ const MultipleChoiceAndOrdering = () => {
...
-        }}
-        css={styles.addOptionButton({
-          currentQuestionType: currentQuestionType as 'multiple_choice' | 'ordering',
-        })}
-      >
-        <SVGIcon name="plus" height={24} width={24} />
-        {__('Add Option', 'tutor')}
-      </button>
+      <div css={styles.addOptionButtonWrapper({ isOrdering: currentQuestionType === 'ordering' })}>
+        <Button variant="text" onClick={handleAddOption} buttonContentCss={styles.addOptionButton}>
```
Comment (sazedul-haque) [inline assets/react/v3/entries/course-builder/components/curriculum/question-types/MultipleChoiceAndOrdering.tsx:264]: Here too

File: assets/react/v3/shared/components/fields/quiz/FormFillinTheBlanks.tsx [hunk 4]
```diff
@@ -109,13 +109,12 @@ const FormFillInTheBlanks = ({ field }: FormFillInTheBlanksProps) => {
               </div>
             }
           >
-            <div css={styles.optionInputWrapper}>
+            <div css={[styleUtils.optionInputWrapper, css`gap: ${spacing[16]};`]}>
```
Comment (sazedul-haque) [inline assets/react/v3/shared/components/fields/quiz/FormFillinTheBlanks.tsx:112]: Keep the `optionInputWrapper` here and use `styleUtils` there.

File: assets/react/v3/shared/utils/style-utils.ts [hunk 5]
```diff
@@ -485,4 +488,109 @@ export const styleUtils = {
       outline-offset: 1px;
     }
   `,
+  optionCheckButton: css`
```
Comment (sazedul-haque) [inline assets/react/v3/shared/utils/style-utils.ts:491]: How about making an atom?

**Review by sazedul-haque — CHANGES_REQUESTED** [review]

File: assets/react/v3/entries/course-builder/components/curriculum/question-types/ImageAnswering.tsx (hunk 1 above)
Comment (b-l-i-n-d) [reply assets/react/v3/entries/course-builder/components/curriculum/question-types/ImageAnswering.tsx:190]: Even if I pass the icon as props, I still need to use custom styles to match color of text.

File: assets/react/v3/shared/utils/style-utils.ts (hunk 5 above)
Comment (b-l-i-n-d) [reply assets/react/v3/shared/utils/style-utils.ts:491]: Wouldn't it be overkill, `optionCheckButton` is only used twice in this whole project?

**Pull request merged**